hdp25/build.properties

```diff
@@ -0,0 +1,42 @@
+impl.title=Pentaho Hadoop HDP 2.5.x Shim
+impl.productID=pentaho-hadoop-shims-hdp25
+ivy.artifact.id=pentaho-hadoop-shims-hdp25
+ivy.artifact.group=pentaho
+project.revision=7.1-SNAPSHOT
+package.id=pentaho-hadoop-shims-hdp25-package
+package.root.dir=hdp25
+oss-license.filename=PentahoHadoopShim_hdp25_OSS_Licenses.html
+
+# Common parts integrating
+common.modern.shim.include=true
+common.hadoop.shim.version=1.0
+common.hbase.shim.version=1.1
+common.pig.shim.version=1.1
+hbase.generation=1.0
+
+dependency.kettle.revision=7.1-SNAPSHOT
+dependency.pentaho-hdfs-vfs.revision=7.1-SNAPSHOT
+dependency.oss-licenses.revision=7.1-SNAPSHOT
+
+dependency.xstream.revision=1.4.2
```

Comment:
We need to try and see if we can use 1.4.7 as versions before that have a security vulnerability